src/proposals/proposal.cpp

```diff
@@ -183,8 +183,15 @@ namespace hypha
         float quorumFactor = m_daoSettings->getOrFail<int64_t>(VOTING_QUORUM_FACTOR_X100) / 100.0f;
         float alignmentFactor = m_daoSettings->getOrFail<int64_t>(VOTING_ALIGNMENT_FACTOR_X100) / 100.0f;
 
-        hypha::voice::stats stats_t(voiceContract, voiceToken.symbol.code().raw());
-        auto stat_itr = stats_t.find(voiceToken.symbol.code().raw());
+        hypha::stats statstable(voiceContract, voiceToken.symbol.code().raw());
+        auto stats_index = statstable.get_index<name("bykey")>();
+
+        auto stat_itr = stats_index.find(
+            currency_stats::build_key(
+                name(""), // use tenant here (as a name)
```

Comment:
Wouldn't you be able to get the tenant from the proposal / document that was being voted on?

Reply:
That's where I got it from in the end. The proposal object has a reference to the dao setting.